pypeit/core/extract.py

```diff
@@ -1497,8 +1502,18 @@ def objfind(image, thismask, slit_left, slit_righ, inmask=None, fwhm=3.0, use_us
     else:
         nobj_reg = 0
 
+    # Set objfind QA filename
```

Comment:
My suggestion is to not pass in this dictionary and synthesise the filename here as you are doing. Instead run the qa.set_qa_filename code in the routine that calls this core objfind method to generate the filename.

Reply:
Done